### PR TITLE
Update: Shopify DOC

### PR DESCRIPTION
**Pull request by aureate-labs-team**
### Related Issues

closes #

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- I tested manually my code and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme
- [ ] I have written test cases for my code

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

**Comment by atlantis-vsf** [comment]
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs  or, to disable this message, set --silence-fork-pr-errors

File: packages/core/docs/shopify/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
-# Vue Storefront for Shopify
+# Shopify integration for Vue Storefront 2
 
-This is a documentation for Vue Storefront2 integration with Shopify.
+:::tip There's more
+This page only contains the documentation specific to shopify integration. For more information, see the [Vue Storefront documentation](https://docs.vuestorefront.io/v2/).
```
Comment (igorwojciechowski) [inline packages/core/docs/shopify/README.md:4]:
```suggestion
This page only contains the documentation specific to Shopify integration. For more information, see the [Vue Storefront documentation](https://docs.vuestorefront.io/v2/).
```

File: packages/core/docs/shopify/README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+:::
 
-This integration is currently a **work in progress** and not ready for production usage, but yes its development-ready.
+[Shopify](https://www.vuestorefront.io/shopify) integration for Vue Storefront is open-source and maintained by the superheros from the [Aureate labs](https://aureatelabs.com/).
```
Comment (igorwojciechowski) [inline packages/core/docs/shopify/README.md:7]:
```suggestion
[Shopify](https://www.vuestorefront.io/shopify) integration for Vue Storefront is open-source and maintained by the superheroes from the [Aureate labs](https://aureatelabs.com/).
```

File: packages/core/docs/shopify/README.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+
+Before diving into it, if you'd like to experience our live [demo](https://shopify-pwa.aureatelabs.com/).
```
Comment (igorwojciechowski) [inline packages/core/docs/shopify/README.md:9]:
```suggestion
If you want to preview our product, feel free to visit our [demo](https://shopify-pwa.aureatelabs.com/) application.
```

File: packages/core/docs/shopify/authorization-strategy.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Authorization
+
+We are following [HTTP Authorization guide](https://shopify.dev/apps/auth) from Shopify documentation and using their library - [JS buy SDK](https://github.com/Shopify/js-buy-sdk). A token is stored in the cookie as we access it both the client and server side.
```
Comment (igorwojciechowski) [inline packages/core/docs/shopify/authorization-strategy.md:3]:
```suggestion
We are following [HTTP Authorization guide](https://shopify.dev/apps/auth) from Shopify documentation and using their library - [JS buy SDK](https://github.com/Shopify/js-buy-sdk). A token is stored in the cookie as we access it on both the client and server-side.
```

File: packages/core/docs/shopify/getting-started.md [hunk 5]
```diff
@@ -1,8 +1,18 @@
...
 
-To get started  initialize the project from [Vue Storefront CLI](https://docs.vuestorefront.io/v2/general/installation.html).
+## Before dive into the Shopify integration with Vue Storefront 2
 
-Then set up your credentials in `middleware.config.js`
+
+If you [generated your project from our CLI](/general/installation.html), your shop will be connected to our [demo store](https://shopify-pwa.aureatelabs.com/).
+
+
+If you haven't yet generated your project and just to play with Vue Storefront integration with Shopify and understand its capabilities, the first thing you should do after setting it up is, changing the credentials to connect to your Shopify store.
```
Comment (igorwojciechowski) [inline packages/core/docs/shopify/getting-started.md:9]:
```suggestion
If you haven't yet generated your project and just to play with Vue Storefront integration with Shopify and understand its capabilities, the first thing you should do after setting it up is changing the credentials to connect to your Shopify store.
```

**Comment by atlantis-vsf** [comment]
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs  or, to disable this message, set --silence-fork-pr-errors

**Review by igorwojciechowski — APPROVED** [review]

**Pull request merged**